utils/datasets/cifar10_dataset.py

```diff
@@ -0,0 +1,386 @@
+# Copyright 2021, Google LLC.
```

Comment:
This file is also in the github currently. Maybe the diff isn't up to date (not an expert on Github reviewing)?

Reply:
There were a couple of merge conflicts in these files because of the spacing, so I had to modify them to resolve the conflicts. I'm not very sure how to go about it either, any suggestions?